modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/wal/FileWriteAheadLogManager.java

```diff
@@ -521,6 +525,9 @@ public void setFileIOFactory(FileIOFactory ioFactory) {
 
                         dispatcher.registerProperty(cdcDisabled);
                     });
+
+                if (dsCfg.getMaxCdcBufferSize() > 0)
```

Comment:
May be is will be more clear to rewrite this as `if (dsCfg.getCdcConsumer() != null)` and have some reasonable deafult for buffer size?